kidviz/forms.py

```diff
@@ -121,6 +121,10 @@ def __init__(self, data=None, files=None, auto_id='id_%s', prefix=None,
                          empty_permitted=empty_permitted, instance=instance,
                          use_required_attribute=use_required_attribute)
 
+        self.fields['video_notes'].widget.attrs['accept'] = 'video/*'
```

Comment:
I wonder if we could initialize them in `widgets`, a few lines above, or is there a reason that those need to be initialized only after the object already exists, in `__init__`?